src/schema/v1/ecommerce/types/enums/payment_method_enum.ts

```diff
@@ -0,0 +1,27 @@
+import { GraphQLEnumType } from "graphql"
+
+export const PaymentMethodEnum = new GraphQLEnumType({
```

Comment:
Are we really need this enum?

Reply:
Do you know another way to define it? I followed the project convention for enums.

Reply:
When Exchange schema is merged into Metaphysics, we [rename and prefix](https://github.com/artsy/metaphysics/blob/ca3ca865c173d956c551a60db9d70d371c75cd9e/src/lib/stitching/exchange/schema.ts#L27-L29) the types with `Commerce...`. So the Exchange payment method enum is [already in MP](https://github.com/artsy/metaphysics/blob/ca3ca865c173d956c551a60db9d70d371c75cd9e/_schemaV2.graphql#L4710-L4725) (for free!) as `CommercePaymentMethodEnum` and that's what we'll use for MP. So I think this addition is not needed.

Reply:
good to know, but what about v1? I think that is needed for v1

Reply:
We want to deprecate v1 and I believe we have migrated all BNMO related code to use v2. So I think we can ignore updating v1 schema here (which can be useful speeding up the deprecation). If I'm wrong, we'll find out! 😛

Reply:
Do you mean omitting `_schema` changes?